AsyncDisplayKit/AsyncDisplayKit.h

```diff
@@ -51,8 +51,11 @@
 #import <AsyncDisplayKit/ASStackLayoutSpec.h>
 
 #import <AsyncDisplayKit/_ASAsyncTransaction.h>
-#import <AsyncDisplayKit/_ASAsyncTransactionContainer+Private.h>
 #import <AsyncDisplayKit/_ASAsyncTransactionGroup.h>
+#import <AsyncDisplayKit/_ASDisplayView.h>
+#import <AsyncDisplayKit/ASDisplayNode+Beta.h>
```

Comment:
We need to avoid including the +Beta ones here. They are designed to require being explicitly included by the developer, so that they realize they're relying on beta functionality.

There should not be a reason for developers to rely on _ASDisplayView, so this should also not be in the overall .h header. I am actually surprised to see the transaction and transaction group headers here, but we can skip changing that for now, because they are already there.

I only took a very quick look at this, haven't finished reading your description, so let me know if there is a reason we must include the beta headers here… We may need to look at restructuring it a little bit if there is no way around this. But it is important that those methods are not generally available, so maybe we need to add "beta" to their method names or something else.